blog/_posts/2017-03-01-zinc-blog-post.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: blog
+post-type: blog
+by: Jorge Vicente Cantero
+title: Speed up compile times with Zinc 1.0
+---
+# Speed up compile times with Zinc 1.0
+
+Zinc is the incremental compiler for Scala. Most Scala developers use it daily
+without noticing -- it's embedded in key build tools like sbt, CBT and pants.
+  
+Zinc has one end goal: to make your compilation times faster without sacrificing
+correctness. When you change a source file, Zinc analyses the dependencies of
+your code and compiles the minimum subset of source files affected by your
+change. The generated code should be identical to the output of a clean compile.
+  
+The new version of Zinc is 1.0 and it has been under development for several
```

Comment:
Maybe we should mention here that old Zinc was republished sbt code, but the new Zinc has its own repo.